source/faq.txt

```diff
@@ -47,16 +47,7 @@ POJOs
 Do I have to specify an ID field value myself?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Imagine you have the following entity:
-
-.. code-block:: java
-
-   public class Entity {
-      public final ObjectId id;
-      public String value;
-
-      public Entity(){ }
-   }
+No, the PojoCodec automatically generates an ``ObjectId``.
```

Comment:
```suggestion
No, the PojoCodec automatically generates an ObjectId.
```

Question: Should ObjectId not be monospaced? [On the ObjectID manual page](https://docs.mongodb.com/manual/reference/method/ObjectId/) it seems like ObjectId is not monospaced.